builtin/logical/ssh/path_config_issuers.go

```diff
@@ -59,6 +59,12 @@ func pathConfigIssuers(b *backend) *framework.Path {
 }
 
 func (b *backend) pathReadDefaultIssuerHandler(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	txRollback, err := logical.StartTxStorage(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	defer txRollback()
+
```

Comment:
Interesting, I did not add a transaction here on purpose as this is only read operation. The same applies for all other operations where we have only reads.

Reply:
If it's a single read it shouldn't be necessary but for multiple reads and consistency, it would be nice.

Reply:
Ok, I understand. Wrapping multiple reads in a transaction, we make sure that all those reads see the same snapshot of the data. There shouldn't be many endpoints with a single read so we should be fine in using transactions everywhere.